pkg/dt/fdt.go

```diff
@@ -411,3 +413,35 @@ func (fdt *FDT) Write(f io.Writer) (int, error) {
 	_, err := w.Write(strs)
 	return w.N, err
 }
+
+// NodeByName finds a node by name.
+func (fdt *FDT) NodeByName(name string) (*Node, bool) {
+	return fdt.RootNode.Find(func(n *Node) bool {
+		return n.Name == name
+	})
+}
+
+// Bytes returns a []byte for the "data" property of a node,
+// or an error if the node has no data property.
+func (fdt *FDT) Bytes(name string) ([]byte, error) {
+	n, ok := fdt.NodeByName(name)
+	log.Printf("Bytes: nodebyname for %q is %s", name, fdt)
+	if !ok {
+		return nil, os.ErrExist
+	}
+	p, ok := n.LookProperty("data")
+	if !ok {
+		return nil, fmt.Errorf("%s does not have a data property", n)
+	}
+	return p.Value, nil
+}
+
+// Reader returns a Reader for the data property of a Node with name 'name',
+// or an error if the node is not found or the node has no data property
+func (fdt *FDT) Reader(name string) (io.ReaderAt, error) {
```

Comment:
Same as previous comment. Move to `fit` package or call `FitBytesReader`.

Reply:
same argument. data tickles my memory as being a common property name?